Jellyfin.Server.Implementations/Item/BaseItemRepository.cs

```diff
@@ -1009,20 +1010,57 @@ public static BaseItemDto DeserialiseBaseItem(BaseItemEntity baseItemEntity, ILo
 
         using var context = _dbProvider.CreateDbContext();
 
-        var query = TranslateQuery(context.BaseItems.AsNoTracking(), context, filter);
+        var innerQueryFilter = new InternalItemsQuery(filter.User)
+        {
+            ExcludeItemTypes = filter.ExcludeItemTypes,
+            IncludeItemTypes = filter.IncludeItemTypes,
+            MediaTypes = filter.MediaTypes,
+            AncestorIds = filter.AncestorIds,
+            ItemIds = filter.ItemIds,
+            TopParentIds = filter.TopParentIds,
+            ParentId = filter.ParentId,
+            IsAiring = filter.IsAiring,
+            IsMovie = filter.IsMovie,
+            IsSports = filter.IsSports,
+            IsKids = filter.IsKids,
+            IsNews = filter.IsNews,
+            IsSeries = filter.IsSeries
+        };
+        var innerQuery = PrepareItemQuery(context, filter)
```

Comment:
Should this be using `innerQueryFilter`?

Reply:
No filter is correct. The Prepare method only configures the navigation properties and the filter is used for loading images. That is not set in innerQueryFilter

Reply:
Ok, then `innerQueryFilter` should be removed as it is unused

Reply:
reused the inner query filter now